resources/lib/scrobbler.py

```diff
@@ -126,14 +126,14 @@ def playbackStarted(self, data):
             elif utilities.isEpisode(self.curVideo['type']):
                 if 'id' in self.curVideo:
                     episodeDetailsKodi = kodiUtilities.getEpisodeDetailsFromKodi(self.curVideo['id'], ['showtitle', 'season', 'episode', 'tvshowid', 'uniqueid', 'file', 'playcount'])
-                    tvdb = episodeDetailsKodi['imdbnumber']
+                    ids = utilities.parseIdToTraktIds(episodeDetailsKodi['imdbnumber'], self.curVideo['type'])[0]
```

Comment:
well it's not that important, both works and is kind of smiliar. Both names are not 100% true.